lib/ddtrace/contrib/rails/framework.rb

```diff
@@ -26,7 +26,8 @@ module Framework
           tracer: Datadog.tracer,
           debug: false,
           trace_agent_hostname: Datadog::Writer::HOSTNAME,
-          trace_agent_port: Datadog::Writer::PORT
+          trace_agent_port: Datadog::Writer::PORT,
+          env: ::Rails.env
```

Comment:
Thanks for all this detailed feedback, FYI, we're trying to address this issue in #106